index.coffee

```diff
@@ -8,15 +8,20 @@ exec = require('child_process').exec
 gutil = require 'gulp-util'
 through = require 'through2'
 
-module.exports = (options = {}) ->
+module.exports = (options = {bundle:true}) ->
 
```

Comment:
PLZ make this happen. I am modifying the node_module manually every install cause of the bundle thing.